### PR TITLE
[Vision Glass] Correctly update applicationId for global App Gallery

### PR DESCRIPTION
**Pull request by felipeerias**
To release Wolvic in the global App Gallery, we need to use this application ID:

    com.igalia.wolvic.world.visionglass

This change replaces an earlier one that was not working as expected.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -436,6 +436,18 @@ android {
         
     }
 
+    androidComponents {
+        onVariants(selector().withBuildType("release")) { variant ->
```
Comment (svillar) [inline app/build.gradle:440]: Not sure we want to limit it to release builds. I think this is also useful for testing, we might want to have both mainland china and world versions.

Also we already have some `configureEach` loops for finetuning variants, couldn't we integrate this into any of those?

**Review by svillar — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (svillar) [reply app/build.gradle:440]: Ping @felipeerias

Comment (felipeerias) [reply app/build.gradle]: @svillar I have updated the PR so the changes apply to both release and debug builds.

Unfortunately the change in application Id needs to happen at a very specific time in the build process so I was not able to include it in our existing loops.

**Review by svillar — APPROVED** [review]
OK, let it be

**Pull request merged**